### PR TITLE
#2: Toggle doors opening and closing

### PR DESCRIPTION
**Pull request by DP94**

### CODE REVIEW

File: Assets/Scripts/Door/Door.cs [hunk 1]
```diff
@@ -0,0 +1,85 @@
+using Mirror;
+using UnityEngine;
+
+public class Door : NetworkBehaviour
+{
+    [SyncVar] [SerializeField] private DoorState _DoorState = DoorState.Closed;
```
Comment (VTVDV) [inline Assets/Scripts/Door/Door.cs:6]: @DP94 Does [SyncVar] indicate it's something that's to be shared between players in multiplayer? :)

Comment (DP94) [reply Assets/Scripts/Door/Door.cs:6]: Yes, it's Mirror's way of synchronising properties across all clients.

Comment (DP94) [reply Assets/Scripts/Door/Door.cs:6]: https://mirror-networking.gitbook.io/docs/guides/synchronization/syncvars

File: Assets/Scripts/Door/Door.cs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    [SyncVar] [SerializeField] private float _doorTimeElapsed;
+
+    [SerializeField]
+    private GameObject _leftDoor;
+    
+    [SerializeField]
+    private GameObject _rightDoor;
+
+    [SerializeField]
+    private bool _zOpen;
```
Comment (VTVDV) [inline Assets/Scripts/Door/Door.cs:22]: What does the z here represent?

Comment (DP94) [reply Assets/Scripts/Door/Door.cs:22]: If it should open on the Z or X axis

Comment (VTVDV) [reply Assets/Scripts/Door/Door.cs:22]: I like the tooltip. :)

**Review by VTVDV — APPROVED** [review]

**Pull request merged**